pkg/test/runner/runner.go

```diff
@@ -128,7 +128,7 @@ func (r *Runner) runFnEval() error {
 	var output string
 	var fnErr error
 	command := run.GetMain()
-	for i := 0; i < r.testCase.Config.RunCount; i++ {
+	for i := 0; i < r.testCase.Config.RunCount(); i++ {
```

Comment:
Should the expected results be tested after each iteration instead of at the end?

Example:

replica = 0
Function: replica++
Expected: replica = 2

This function is not idempotent.

Reply:
Update the implementation and now the result will be checked after each iteration.